vanilla-framework/vanilla-framework
# Update docs/cards page content

## Done
- Updated copy ✏️ 
- Updated demo examples 💯 
- Check content matches copy doc [here](https://docs.google.com/document/d/1ZFRd4z7TsU4oifxgky77337v_1fkE2ak12tGPNivD70/edit#heading=h.ldypsxh84552)

## QA
- Pull code
- Run /docs
  - Check http://0.0.0.0:8104/en/patterns/card
- Pull code
- Run ./run
- Review example
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/card/default/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/card/header/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/card/highlighted/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/card/overlay/

## Screenshot
![Screenshot 2019-04-08 at 10 19 03](https://user-images.githubusercontent.com/17748020/55713359-6a2b0600-59e8-11e9-93b8-702f6ec83912.png)
![Screenshot 2019-04-08 at 10 19 30](https://user-images.githubusercontent.com/17748020/55713360-6ac39c80-59e8-11e9-937e-403516811ef6.png)
